### PR TITLE
Add mathematica unit test file extension

### PR DESCRIPTION
**Pull request by ibnfirnas**
#### Usage in the wild
- GitHub search: https://github.com/search?utf8=%E2%9C%93&q=%22Test%5B%22+OR+%22TestSuite%5B%22+extension%3Amt&type=Code&ref=advsearch&l=&l=
- Google search of GitHub.com: https://www.google.im/search?q=%22Test%5B%22+OR+%22TestSuite%5B%22+filetype:mt+site:github.com

### CODE REVIEW

**Comment by arfon** [comment]
@ibnfirnas - would you mind accepting the GitHub CLA?

**Comment by ibnfirnas** [comment]
@arfon :

- CLA: done
- Search link: strangely, nothing is found with GitHub search, but I get 293 results for a google search of github.com: https://www.google.im/search?q=%22Test%5B%22+OR+%22TestSuite%5B%22+filetype:mt+site:github.com

**Comment by ibnfirnas** [comment]
@arfon: all done

**Comment by arfon** [comment]
:+1: thanks @ibnfirnas. This will be live in the next release of Linguist.

**Comment by ibnfirnas** [comment]
@arfon: Awesome - thanks!

File: samples/Mathematica/TestString.mt [hunk 1]
```diff
@@ -0,0 +1 @@
+Test[1"a" <> "b", "ab", TestID -> "Concat \"a\" and \"b\""]
```
Comment (ibnfirnas) [inline samples/Mathematica/TestString.mt]: Dang - I just noticed a copypasta artifact: `Test[1"a" ` should just be `Test["a" `

**Pull request merged**